charts/app-config/templates/signon-secrets-sync-configmap.yaml

```diff
@@ -7,6 +7,7 @@ data:
     [
       "Account API",
       "Asset Manager",
+      "Chat",
```

Comment:
Added this one for now, if we choose a different name I will change it again

Reply:
We won't need this when https://github.com/alphagov/govuk-helm-charts/pull/2006 is merged (which I imagine will be in the next hour).

Thanks for adding it though.